lib/modules/settings/screens/notifications/athan/athan_prayer_settings_screen.dart

```diff
@@ -156,48 +148,44 @@ class _AthanPrayerSettingsScreenState extends State<AthanPrayerSettingsScreen> {
             PrayerNotificationEvent(
                 widget.prayerType, prayerNotificationSettings),
           );
-      // reschedule notifications with the new settings.
-      // debounce scheduling to avoid scheduling multiple times in a short period of time.
```

Comment:
I think its still good to keep this comment

Reply:
Done